__tests__/propertyVolumeTests.ts

```diff
@@ -0,0 +1,21 @@
+import * as properties from '../src'
+import * as config from 'config'
+
+describe('Azure flex properties module should read from specified mount', () => {
```

Comment:
Drop azure flex from this maybe? as its nothing to do with azure its reading properties from files?

Reply:
can do ...